maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/WorkflowPreProcessorServiceTest.java

```diff
@@ -10,80 +10,126 @@
+        WorkflowRequest request = getWorkflowRequestWithCaseType(INDICTABLE.getCaseType());
+        RepOrderDTO repOrderDTO = TestModelDataBuilder.buildRepOrderDTO(RepOrderStatus.CURR.getCode());
+        UserActionDTO userActionDTO = TestModelDataBuilder.getUserActionDTO();
+        UserSummaryDTO userSummaryDTO = TestModelDataBuilder.getUserSummaryDTO();
+        List<String> validationErrors = List.of(
+            ValidationService.USER_DOES_NOT_HAVE_A_VALID_NEW_WORK_REASON_CODE);
+
+        when(maatCourtDataService.getUserSummary(userActionDTO.getUsername())).thenReturn(userSummaryDTO);
+        doNothing().when(validationService).validate(request, repOrderDTO);
+        doThrow(new CrimeValidationException(validationErrors)).when(validationService).isUserActionValid(userActionDTO, userSummaryDTO);
+
+        assertThrows(CrimeValidationException.class, () -> workflowPreProcessorService.preProcessRequest(request, repOrderDTO, userActionDTO));
+    }
+
     @Test
     void givenApplicationStatusHasChanged_whenWorkflowPreProcessorServiceIsInvoked_thenUpdatedWorkflowRequestWithContributionIsReturned() {
         WorkflowRequest request = getWorkflowRequestWithCaseType(INDICTABLE.getCaseType());
         RepOrderDTO repOrderDTO = TestModelDataBuilder.buildRepOrderDTO(RepOrderStatus.CURR.getCode());
-        when(contributionService.calculate(any()))
-                .thenReturn(getApplicationDTO());
-        workflowPreProcessorService.preProcessRequest(request, repOrderDTO);
+        UserActionDTO userActionDTO = TestModelDataBuilder.getUserActionDTO();
+        UserSummaryDTO userSummaryDTO = TestModelDataBuilder.getUserSummaryDTO();
+
+        when(contributionService.calculate(any())).thenReturn(TestModelDataBuilder.getApplicationDTO());
+        when(maatCourtDataService.getUserSummary(userActionDTO.getUsername())).thenReturn(userSummaryDTO);
+        doNothing().when(validationService).validate(request, repOrderDTO);
+        when(validationService.isUserActionValid(userActionDTO, userSummaryDTO)).thenReturn(true);
+
+        workflowPreProcessorService.preProcessRequest(request, repOrderDTO, userActionDTO);
+
         verify(contributionService).calculate(any());
     }
 
     @ParameterizedTest
     @MethodSource("workflowRequestWithNoApplicationStatusChange")
     void givenApplicationStatusHasNotChanged_whenWorkflowPreProcessorServiceIsInvoked_thenWorkflowRequestIsReturned(WorkflowRequest request, RepOrderDTO repOrderDTO) {
-        workflowPreProcessorService.preProcessRequest(request, repOrderDTO);
+        UserActionDTO userActionDTO = TestModelDataBuilder.getUserActionDTO();
+        UserSummaryDTO userSummaryDTO = TestModelDataBuilder.getUserSummaryDTO();
+
+        when(maatCourtDataService.getUserSummary(userActionDTO.getUsername())).thenReturn(userSummaryDTO);
+        doNothing().when(validationService).validate(request, repOrderDTO);
```

Comment:
Was this comment referring to the verify call of the _ValidationService_ `validate` method? If so the reason I used `doNothing` (which I've now removed anyway) is because that method returns `void` and not `boolean`.